kontrol/kontrol.go

```diff
@@ -372,7 +372,6 @@ func generateToken(aud, username, issuer, privateKey string) (string, error) {
 	tkn.Claims["sub"] = username                                     // Subject
 	tkn.Claims["aud"] = aud                                          // Audience
 	tkn.Claims["exp"] = time.Now().UTC().Add(ttl).Add(leeway).Unix() // Expiration Time
-	tkn.Claims["nbf"] = time.Now().UTC().Add(-leeway).Unix()         // Not Before
```

Comment:
lets make this more general, if `TokenLeeway` is 0, then disable this functionality, otherwise ignoring the parameter would be problematic and implicit for other users.